src/KeyHandlerTest.cpp

```diff
@@ -305,8 +305,8 @@ TEST_F(KeyHandlerTest,
   auto endState = handleKeySequence(keys);
   auto inputtingState = dynamic_cast<InputStates::Inputting*>(endState.get());
   ASSERT_TRUE(inputtingState != nullptr);
-  ASSERT_EQ(inputtingState->composingBuffer, "ㄧˊ");
-  ASSERT_EQ(inputtingState->cursorIndex, strlen("ㄧˊ"));
+  ASSERT_EQ(inputtingState->composingBuffer, "一");
```

Comment:
Test case should be renamed to `ToneMarkThenNonToneComponentResultingInCompositionCase1` or something like that?

Reply:
Done.